lib/ExWebhook/schema/webhook.ex

```diff
@@ -17,6 +17,10 @@ defmodule ExWebhook.Schema.Webhook do
     field(:is_batch, :boolean)
     has_many(:webhook_events, ExWebhook.Schema.WebhookEvent)
 
+    field(:deactivated, :boolean, default: false)
+    field(:deactivated_at, :utc_datetime)
```

Comment:
I think we don't need `deactivated_at`, we can just use `deactivated_at != null`